temporian/core/operators/lag.py

```diff
@@ -84,21 +86,67 @@ def build_op_definition(cls) -> pb.OperatorDef:
 operator_lib.register_operator(LagOperator)
```

Comment:
In "build_op_definition", the duration is defined as a float64. Can you make sure (through all the code) that the build_op_definition matches the implementation (pytype, isintance, etc.). I think, currently, some of the pytype are outdated.